datastream-server/src/main/java/com/linkedin/datastream/server/assignment/PartitionAssignmentStrategyConfig.java

```diff
@@ -54,6 +58,8 @@ public PartitionAssignmentStrategyConfig(Properties config) {
     int cfgMaxParitionsPerTask = props.getInt(CFG_MAX_PARTITION_PER_TASK, 0);
     int cfgPartitionsPerTask = props.getInt(CFG_PARTITIONS_PER_TASK, 0);
     int cfgPartitionFullnessThresholdPct = props.getIntInRange(CFG_PARTITION_FULLNESS_THRESHOLD_PCT, 0, 0, 100);
+    int cfgTaskCapacityMBps = props.getInt(CFG_TASK_CAPACITY_MBPS, 0);
```

Comment:
Just wondering, if we should get the default values to this class and get rid of optional.

The counter reasoning to this change request can be: in the future, we may have some class that does not need the default value. But, we can move to Optional at that time. Right now, even though we pass optional, the constructor of the class, sets the default value if Optional is empty.

Reply:
I tried to stick to the way other configs were implemented. If we want to change this, we might want to address the older ones too to keep things consistent.